v22.2/create-changefeed.md

```diff
@@ -28,8 +34,20 @@ To create a changefeed, the user must be a member of the `admin` role or have th
 Parameter | Description
 ----------|------------
 `table_name` | The name of the table (or tables in a comma separated list) to create a changefeed for.<br><br>**Note:** Before creating a changefeed, consider the number of changefeeds versus the number of tables to include in a single changefeed. Each scenario can have an impact on total memory usage or changefeed performance. See [Create and Configure Changefeeds](create-and-configure-changefeeds.html) for more detail.
-`sink` | The location of the configurable sink. The scheme of the URI indicates the type. For more information, see [Sink URI](#sink-uri) below.
-`option` / `value` | For a list of available options and their values, see [Options](#options) below.
+`sink` | The location of the configurable sink. The scheme of the URI indicates the type. For more information, see [Sink URI](#sink-uri).
+`option` / `value` | For a list of available options and their values, see [Options](#options).
+
+### Changefeed expression parameters
+
+The [Changefeed Expressions](changefeed-expressions.html) page provides detail on the functionality, syntax, and use cases for changefeeds created with expressions. 
+
+Parameter | Description
+----------|------------
+`sink` | The location of the configurable sink. The scheme of the URI indicates the type. For more information, see [Sink URI](#sink-uri).
+`option` / `value` | For a list of available options and their values, see [Options](#options).
+`target_list` | The columns to emit data from.
```

Comment:
This line looks a little off? In my head the syntax looks something like

`CREATE CHANGEFEED [WITH options] [INTO sink] AS SELECT projection FROM target [WHERE predicate]`

Reply:
Hmmm, so I ran all of expressions as `CREATE CHANGEFEED INTO sink WITH options AS SELECT....` 

With the option clause first, I receive a syntax error.